docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    intersect_point = (-117.599330, 35.769500)
</pre><p>I'm not sure there's much utility in defining this tuple, since it's just being hard-coded and only used once. I think I'd just hard-code it in the WKT string</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/hyp3-docs/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>